docs/installation.rst

```diff
@@ -9,11 +9,11 @@ Local
 =====
 Released versions of HeuDiConv are available on `PyPI <https://pypi.org/project/heudiconv/>`_
 and `conda <https://github.com/conda-forge/heudiconv-feedstock#installing-heudiconv>`_.
-If installing through ``PyPI``, eg::
+To install through ``PyPI``, run::
 
     pip install heudiconv[all]
 
-On Debian-based systems we recommend using `NeuroDebian <http://neuro.debian.net>`_
+On Debian-based systems, we recommend using `NeuroDebian <http://neuro.debian.net>`_
```

Comment:
chatgpt said we need it, poor mortal Yarik surrenders to the wisdom of Isaac & ChatGPT